blockchain/types/anchoring_data.go

```diff
@@ -62,3 +67,26 @@ func NewAnchoringDataType0(block *Block, period *big.Int, txCount *big.Int) (*An
 	}
 	return &AnchoringData{AnchoringDataType0, encodedCCTxData}, nil
 }
+
+// DecodeAnchoringTx decodes an anchoring transaction used by main and sub bridges.
+func DecodeAnchoringTx(data []byte) (common.Hash, *big.Int, error) {
```

Comment:
Why don't you use "DecodeAnchroingData"?

Reply:
Fixed.